tardis/tests/integration_tests/test_integration.py

```diff
@@ -28,6 +30,7 @@ def setup(self, request, reference, data_path, atomic_data_fname):
         self.name = data_path['setup_name']
 
         self.config_file = os.path.join(data_path['config_dirpath'], "config.yml")
+        self.config_dict = yaml.load(self.config_file)
```

Comment:
Why did you change it anyway? To me it seems like `Configuration.from_yaml(...)` does everything we want.

We might even go so far as to use `tardis.run_tardis(config)`. Any reason to not use this?